python/cuml/dask/neighbors/nearest_neighbors.py

```diff
@@ -40,12 +40,33 @@ def _func_get_i(f, idx):
 class NearestNeighbors(BaseEstimator):
     """
     Multi-node Multi-GPU NearestNeighbors Model.
+
+    Parameters
+    ----------
+    n_neighbors : int (default=5)
+        Default number of neighbors to query
+    batch_size: int (optional, default 1024)
+        Maximum number of queries processed at once. This parameter can
+        greatly affect the throughput of the algorithm.
```

Comment:
```suggestion
        greatly affect the throughput of the algorithm. The optimal setting of this value will vary for different layouts index to query ratios, but it will require `batch_size * n_features * 4` bytes of additional memory on each worker hosting index partitions.
```